src/reader/ns_reader.rs

```diff
@@ -48,6 +50,12 @@ impl<R> NsReader<R> {
     pub fn config_mut(&mut self) -> &mut Config {
         self.reader.config_mut()
     }
+
+    /// Returns all the prefixes currently declared except the default xml: and xmlns: namespaces.
+    #[inline]
+    pub fn prefixes(&self) -> impl Iterator<Item = (PrefixDeclaration, Namespace)> {
```

Comment:
What if you wish to store returned iterator in the struct? Is it possible? If yes, you should name returned iterator explicitly

Reply:
Yes! Done!